assets/_globalCss.css

```diff
@@ -203,3 +209,4 @@ input[type='password'][disabled] {
   cursor: not-allowed;
   border-color: var(--color-disabled);
 }
+*/
```

Comment:
Can you actually remove the old code once you submit this PR as a non-draft one?